OmsAgent/Fairfax/omsagent_ff.py

```diff
@@ -731,7 +731,7 @@ def log_and_exit(hutil, operation, exit_code = 1, message = ''):
     if hutil is not None:
         hutil.do_exit(exit_code, operation, exit_status, str(exit_code), message)
     else:
-        update_status_file(operation, exit_code, exit_status, message)
+        update_status_file(operation, str(exit_code), exit_status, message)
```

Comment:
What I mean is only VM extension specific error should be logged here. Any error code that is thrown by the bundle already, should be maintained in the agent code base only.